pkg/virt-handler/cache/cache.go

```diff
@@ -52,7 +52,7 @@ func (icp *iterableCheckpointManager) ListKeys() []string {
 		return []string{}
 	}
 
-	keys := []string{}
+	var keys []string
```

Comment:
In your commit message, can you explain why this is a good idea?
I'm not a good GoLang developer, so I don't understand the benefit here.

Reply:
Yes, definitely! Thanks for pointing that out.

I've added the following:
```
An empty slice is different than a nil slice
in that a nil slice captures 0 resources, while
an empty map demands allocating a small amount of
memory.

Generally speaking, unless there's a reason for
the map to never be nil, it's better to not allocate
memory unless we actually need it.
```